### ISSUE TITLE
UnicodeDecodeError: 'ascii' codec can't decode byte 0xd1 in position 74752: ordinal not in range(128)

### ISSUE THREAD

**Issue by ericclemmons** [devstructure/blueprint#17]
```
$ sudo blueprint create test -S
# [blueprint] searching for software built from source
# [blueprint] searching for PEAR/PECL packages
# [blueprint] searching for configuration files
# [blueprint] searching for Python packages
# [blueprint] searching for apt packages
# [blueprint] searching for Ruby gems
Reinitialized existing Git repository in /home/eric/.blueprints.git/
Traceback (most recent call last):
  File "/usr/local/bin/blueprint-create", line 58, in <module>
    filename = getattr(b, options.generate)().dumpf()
  File "/usr/local/lib/python2.6/dist-packages/blueprint/sh.py", line 57, in dumpf
    f2.write(content)
  File "/usr/lib/python2.6/codecs.py", line 691, in write
    return self.writer.write(data)
  File "/usr/lib/python2.6/codecs.py", line 351, in write
    data, consumed = self.encode(object, self.errors)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xd1 in position 74752: ordinal not in range(128)
```

Is there some file I can provide for easier debugging?

**Comment by consiliens** [comment]
```
$ sudo blueprint create --chef virt
# [blueprint] searching for Ruby gems
# [blueprint] searching for software built from source
# [blueprint] searching for apt packages
# [blueprint] searching for configuration files
# [blueprint] searching for Python packages
# [blueprint] searching for PEAR/PECL packages
Initialized empty Git repository in /home/.blueprints.git/
Traceback (most recent call last):
  File "/usr/local/bin/blueprint-create", line 58, in <module>
    filename = getattr(b, options.generate)().dumpf()
  File "/usr/local/lib/python2.6/dist-packages/blueprint/chef.py", line 118, in dumpf
    f.write(resource.content)
  File "/usr/lib/python2.6/codecs.py", line 691, in write
    return self.writer.write(data)
  File "/usr/lib/python2.6/codecs.py", line 351, in write
    data, consumed = self.encode(object, self.errors)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xd1 in position 47616: ordinal not in range(128)
```

Just tried blueprint and encountered the same error.

**Comment by gurgeous** [comment]
I hit this too. Would love to try the tool someday, thanks.

**Comment by danielsoneg** [comment]
This is the problem I was addressing in #16 - the changes from #9 attempt to save a .tar file using the codecs module, which causes this error. I've got a pull request with a fix.

**Comment by rcrowley** [comment]
This is a duplicate of #16 and is closed as of 1759813b1f6554e0e5e7b17105959a723f126c06.
